### ISSUE TITLE
SVG very small

### ISSUE THREAD

**Issue by mbabini** [RealFaviconGenerator/realfavicongenerator#520]
Hello

I suddenly have the problem that my SVG files are only displayed very small. You can see it clearly with the Apple Touch icon: The icon only covers about a quarter of the available space.

![image](https://github.com/user-attachments/assets/09d587f6-b498-4369-ad57-93f1ceaf82e6)

I also tried it with older SVG files that I used some time ago without any problems. The older files are now also only displayed in a smaller size in the preview.

Thank you very much for your help!

![favicon_b](https://github.com/user-attachments/assets/02098657-e884-417c-ae7e-f8d28344225f)

**Comment by phbernard** [comment]
When I review this issue with desktop Chrome and I click the icon at the bottom of your message, Chrome opens it and shows:

![image](https://github.com/user-attachments/assets/bd0043c8-ebd3-4230-8be6-aec0fbf258e7)

Same with Firefox and Edge. Do you have the same behavior with your original image?

**Comment by mbabini** [comment]
Salut Philippe

Thank you very much for your answer. The original file can be displayed with Chrome without errors. I have generated it with Affinity Designer. You can find the original file [here](https://www.sennenberg.ch/images/favicon_b.svg).

Thank you!

**Comment by phbernard** [comment]
Got it!

Not a fix, but at least a clue: this is apparently due to the `width="100%" height="100%"` in the top-level `svg` markup. As soon as I remove them:

![image](https://github.com/user-attachments/assets/82e7d7c7-7023-4b85-9348-fc2d7fd392ba)

Not sure why these attributes are needed in the first place, but I suppose RealFavicon should support this case.

**Comment by mbabini** [comment]
You are right, if I adjust the SVG with an editor and remove the width and height, it works! Thank you very much for your help!

**Comment by jjnxpct** [comment]
I have the same issue., Exporting the SVG from Affinity Designer adds the 100% width and height that causes issues with the realfavicon site. Removing the width and height works and we can generate the favicons with that modification. But may there can be a fix for this on the website so we don't have to manually 'fix' our SVG file?
